### PR TITLE
feat: adds pods to pvc view, make swagger more discoverable

### PR DESCRIPTION
**Pull request by UncleGedd**
## Description

- Adds pods to PVC view to match Lens
- Makes Swagger more discoverable by redirecting `/swagger` -> `/swagger.index.html`
- Simplify `expectEqualIgnoringFields` with `lodash` deep copy functions
- Updates Svelte version to address Dependabot vulnerability alert

Runtime:
Screenshot 2024-09-01 at 11 21 26 AM

Lens:
Screenshot 2024-09-01 at 11 08 56 AM

### CODE REVIEW

File: ui/src/lib/features/k8s/storage/persistentvolumeclaims/component.test.ts [hunk 1]
```diff
@@ -26,13 +28,25 @@ suite('PersistentVolumeClaim Component', () => {
...
 
+  const urlAssertionMock = vi.fn().mockImplementation((url: string) => {
+    console.log(url)
+  })
+
+  vi.stubGlobal(
+    'EventSource',
+    vi
+      .fn()
+      // metrics EventSource is created first in createStore()
```
Comment (TristanHoladay) [inline ui/src/lib/features/k8s/storage/persistentvolumeclaims/component.test.ts:46]: nit
```suggestion
      // pods EventSource is created first in createStore()
```

**Comment by decleaver** [comment]
Getting a 401 when I click on the pods in the pvc view when api token auth is enabled.... Curious is this a personal problem ?

File: ui/src/lib/features/k8s/storage/persistentvolumeclaims/store.ts [hunk 2]
```diff
@@ -1,35 +1,84 @@
...
   capacity: string
   pods: string[]
-  status: string
+  status: { component: typeof Status; props: { status: string } }
 }
 
 export type Columns = ColumnWrapper<Row>
 
 export function createStore(): ResourceStoreInterface<Resource, Row> {
   const url = `/api/v1/resources/storage/persistentvolumeclaims?dense=true`
```
Comment (decleaver) [inline ui/src/lib/features/k8s/storage/persistentvolumeclaims/store.ts:22]: any benefit to using fields vs dense here?

Comment (decleaver) [reply ui/src/lib/features/k8s/storage/persistentvolumeclaims/store.ts:22]: created follow on issue: https://github.com/defenseunicorns/uds-runtime/issues/275

**Comment by decleaver** [comment]
> Getting a 401 when I click on the pods in the pvc view when api token auth is enabled.... Curious is this a personal problem ?

Created and working on follow on issue https://github.com/defenseunicorns/uds-runtime/issues/274

**Review by decleaver — APPROVED** [review]
Looks good, created follow on issues to address comments

**Pull request merged**